grafana/grafana/base/jupyterhub-dashboards/jupyterhub-usage-dashboard.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     app: grafana
 spec:
+  json: ""
   configMapRef:
```

Comment:
Why is this field needed ```json```

Reply:
I already removed it.

It was needed in 4.0.1 because the CRD didn't have the `omitempty` flag. Since the new 4.0.2 version fixed it, it is not needed anymore (https://github.com/grafana-operator/grafana-operator/pull/606).